Application/FoxFlss/data/Makefile.am

```diff
@@ -1,87 +1,33 @@
-# Directory for user-enabled tools
+ACLOCAL_AMFLAGS = -I m4
+
 FoxFlss_dir = /opt/foxconn
-FoxFlss_datadir = $(FoxFlss_dir)/data
-FoxFlss_userdir = $(FoxFlss_dir)/fcc-unlock.d
-FoxFlss_suspdir = $(FoxFlss_dir)/suspend
+FoxFlss_rfdatadir = $(FoxFlss_dir)/data
+FoxFlss_fccunlockdir = $(FoxFlss_dir)/fcc-unlock.d
+FoxFlss_suspenddir = $(FoxFlss_dir)/suspend
+FoxFlss_modemmanagerservicedir = $(sysconfdir)/systemd/system/ModemManager.service.d
```

Comment:
$(sysconfdir) is $(prefix)/etc, but what is expected is only /etc.

Reply:
I see why when building packages, it would automatically point to `/etc`:
There's a toolset called `dh` (Debian helper), which runs commands for the build environment automatically, and it gives the following commands when running `./configure` as it found out it is a GNU autoconf build environment:

```cd Application && ./configure --build=x86_64-linux-gnu --prefix=/usr --includedir=\${prefix}/include --mandir=\${prefix}/share/man --infodir=\${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --disable-option-checking --disable-silent-rules --libdir=\${prefix}/lib/x86_64-linux-gnu --runstatedir=/run --disable-maintainer-mode --disable-dependency-tracking```

I am wondering whether it is okay for your workflow to add `--sysconfdir=/etc` when you invoke `./configure`.

Reply:
Please refer to the commit:[https://github.com/foxconn-pc/fii_linux/commit/cd3e6b17fbbeb7cdfaa3f730276f27e6bbdf074a](url).
We need the condition of sysconfdir=/etc when ModemManager is successfully called to prevent the system from being unable to connect to the network after entering the suspend state.

Reply:
I made changes, could you check the comment below?